### DISCUSSION TITLE
Can not get extensions to show up

### DISCUSSION THREAD

**Discussion by RunningUtes** [FreshRSS/FreshRSS#6300] [Q&A]
### Describe the bug

I have cloned the official extensions into /extensions/ but can't get seem to get FreshRSS to show anything other then "There are no installed extensions". Installed on Rasp Pi w2 with dietpi OS. 

### To Reproduce

1. Go to account > configuration > extensions
2. See error "There are no installed extensions"

### Expected behavior

extension show listed with switch

### FreshRSS version

1.23.1 

### Environment information

rasp pi w2, dietpi os

**Comment by math-GH** [comment]
Could you please give an insight into your extension folder? How does the folder structure look like there?

**Reply by RunningUtes** [reply]
Tree -d


├── api
├── extensions
│   └── Extensions
│       ├── Docker
│       ├── xExtension-ColorfulList
│       │   └── static
│       ├── xExtension-CustomCSS
│       │   ├── i18n
│       │   │   ├── de
│       │   │   ├── en
│       │   │   └── fr
│       │   └── static
│       ├── xExtension-CustomJS
│       │   ├── i18n
│       │   │   ├── de
│       │   │   ├── en
│       │   │   └── fr
│       │   └── static
│       ├── xExtension-ImageProxy
│       │   └── i18n
│       │       ├── de
│       │       ├── en
│       │       └── fr
│       ├── xExtension-QuickCollapse
│       │   ├── Controllers
│       │   ├── Models
│       │   ├── i18n
│       │   │   ├── de
│       │   │   ├── en
│       │   │   └── fr
│       │   ├── static
│       │   └── views
│       │       └── quickCollapse
│       ├── xExtension-ReadingTime
│       │   └── static
│       ├── xExtension-ShareByEmail
│       │   ├── Controllers
│       │   ├── Models
│       │   ├── i18n
│       │   │   ├── de
│       │   │   ├── en
│       │   │   └── fr
│       │   ├── mailers
│       │   ├── static
│       │   └── views
│       │       ├── shareByEmail
│       │       └── share_mailer
│       ├── xExtension-StickyFeeds
│       │   └── static
│       ├── xExtension-TTRSS_API
│       ├── xExtension-TitleWrap
│       │   └── static
│       ├── xExtension-YouTube
│       │   └── i18n
│       │       ├── de
│       │       ├── en
│       │       └── fr
│       └── xExtension-showFeedID
│           └── static
├── i
├── scripts
│   └── vendor
└── themes
    ├── Alternative-Dark
    │   ├── icons
    │   └── thumbs
    ├── Ansum
    │   ├── icons
    │   └── thumbs
    ├── Dark
    │   ├── icons
    │   └── thumbs
    ├── Dark-pink
    │   └── thumbs
    ├── Flat
    │   └── thumbs
    ├── Mapco
    │   ├── icons
    │   └── thumbs
    ├── Nord
    │   ├── icons
    │   └── thumbs
    ├── Origine
    │   └── thumbs
    ├── Origine-compact
    │   └── thumbs
    ├── Pafat
    │   ├── icons
    │   └── thumbs
    ├── Swage
    │   ├── icons
    │   └── thumbs
    ├── base-theme
    ├── fonts
    └── icons

95 directories

**Reply by Alkarex** [reply]
This seems to be under `./FreshRSS/p/extensions/`, which is wrong. They should be under the existing folder `./FreshRSS/extensions/`
There is a readme in the correct folder https://github.com/FreshRSS/FreshRSS/blob/edge/extensions/README.md
